packages/lib-classifier/src/plugins/drawingTools/experimental/models/tools/FreehandLineTool/FreehandLineTool.js

```diff
@@ -10,14 +10,26 @@ const FreehandLineTool = types
   .actions((self) => ({
     createMark(mark) {
       const newMark = FreehandLine.create(
-        Object.assign({}, mark, { toolType: self.type })
+        { ...mark, toolType: self.type }
       )
+
+      // points assumes [{ x, y }, { x, y }]
+      let points = mark.points || []
+      
+      if (mark.pathX && mark.pathY) {
+        points = mark.pathX.map((x, i) => {
+          return { x: mark.pathX[i], y: mark.pathY[i] }
+        })
+      }
+
+      newMark.initialize(points)
```

Comment:
This could be moved into an `afterAttach` or `afterCreate` hook in the `FreehandLine` model, so that it runs automatically whenever `FreehandLine.create` is called. Here's an example where we automatically timestamp each new classification:
https://github.com/zooniverse/front-end-monorepo/blob/9bad4eadce28314a889fe042b9e6622e2450d6d4/packages/lib-classifier/src/store/Classification/ClassificationMetadata.js#L36-L39